python-rdklib/SQS_ENCRYPTED_KMS/SQS_ENCRYPTED_KMS.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+"""
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  SQS_ENCRYPTED_KMS
+
+Description:
+  Check whether Amazon Simple Queue Service (Amazon SQS) is encrypted
+  with AWS Key Management Service (AWS KMS).
+
+Rationale:
+  Encrypting gives the protection for the content of messages in Amazon SQS queues
+  using keys managed in the AWS Key Management Service (AWS KMS).
+
+Indicative Severity:
+  Medium
+
+Trigger:
+  Configuration Change on AWS::SQS::Queue
+
+Reports on:
+  AWS::SQS::Queue
+
+Rule Parameters:
+  None
```

Comment:
Add KmsArn (check name of parameter in exsiting rule) as an optional parameter to be checked for compliance. A list of key is even better. https://docs.aws.amazon.com/config/latest/developerguide/sagemaker-notebook-kms-configured.html